Add `split-second-stopwatch` exercise (#2547)
exercism/problem-specifications: exercises/split-second-stopwatch/introduction.md

# Introduction

> As an amateur runner, you've never really felt the need for precise time tracking.

Reviewer: ```suggestion
As an amateur runner, you've never really felt the need to track your time precisely.
```

> However, having just enrolled in a local running team, you're now being given precise practice schedules, which are tuned to the second.

Reviewer: ```suggestion
However, having just enrolled in a local running team, you're now following precisely timed training sessions, each one tuned to the second.
```

> You've therefore decided you need to purchase the _split-second stopwatch_, a basic but very precise stopwatch.

Reviewer: ```suggestion
Because of this, you've decided to get the _split-second stopwatch_, a basic but very precise stopwatch.
```

# Introduction

You've always run for the thrill of it — no schedules, no timers, just the sound of your feet on the pavement.
But now that you've joined a competitive running crew, things are getting serious.

> Training sessions are timed to the second, and every split counts.

Reviewer 2: ```suggestion
Training sessions are timed to the second, and every split second counts.
```
This reads awkwardly otherwise.
